spaceship/lib/spaceship/connect_api/provisioning/provisioning.rb

```diff
@@ -68,14 +68,62 @@ def post_bundle_id_capability(bundle_id_id:, capability_type:, settings: [])
                     type: "bundleIds",
                     id: bundle_id_id
                   }
+                },
+                capability: {
+                  data: {
+                    type: "capabilities",
+                    id: capability_type
+                  }
                 }
               }
             }
           }
-
           provisioning_request_client.post("bundleIdCapabilities", body)
         end
 
+        def patch_bundle_id_capability(bundle_id_id:, identifier:, seed_id:, name:, enabled: false, capability_type:, settings: [])
+          body = {
+            data: {
+              type: "bundleIds",
+              id: bundle_id_id,
+              attributes: {
+                identifier: identifier,
+                permissions: {
+                    edit: true,
+                    delete: false
+                },
+                seedId: seed_id,
+                name: name,
+                wildcard: false,
```

Comment:
These don't seem to be needed to patch a bundle id capability. Thoughts on removing these 😇

Reply:
Hey @joshdholtz 

Thanks for reviewing! When you tested with only `teamId` and enabled one capability, did the other enabled capabilities get removed? I noticed that Xcode and the Developer Portal were setting the `delete` field in the `permissions` attribute to `true` and would pass in the entire set of enabled capabilities every time something was changed. The reason I kept this field is to ensure that nothing gets deleted if the user only modifies a certain subset of capabilities. If you saw that the default behavior without this attribute is to preserve the already enabled fields, then I can remove this one. Will also remove the other unnecessary entries.

Reply:
@ShyamLad Yup yup, they still there! Everything still works. Those attributes that the ones on the "bundle id" itself so we don't really want to change of those. We are only interested in changing the relationship 🤷‍♂️

Reply:
@joshdholtz Perfect! Will remove this, update the tests, and request a review from you when its completed

Reply:
fixed!